src/components/tuTable/tuTable.vue

```diff
@@ -34,8 +34,9 @@
 		>
 			<table class="tu-table__element" ref="tableElement">
 				<thead ref="thead" class="tu-table__thead">
+					<slot v-if="$slots.thead" name="thead" />
```

Comment:
Why are there changes in table for a Calendar component PR ?
If so, what are these changes for ?

Reply:
i was working on the slots thing in calendar so was refering the tutable component so while doing that i missed something will fix that